include/swift/AST/Attr.h

```diff
@@ -616,6 +616,10 @@ enum class PlatformAgnosticAvailabilityKind {
   /// The declaration is available in some but not all versions
   /// of Swift, as specified by the VersionTuple members.
   SwiftVersionSpecific,
+  /// The declaration is available in some but not all versions
+  /// of SwiftPM's PackageDescription library, as specified by
+  /// the VersionTuple members.
+  SwiftPMManifestVersionSpecific,
```

Comment:
It seems like you should either consistently call this "SwiftPM manifest" or "PackageDescription". Right now it seems like the only place it's called "PackageDescription" is when you actually write it, and I don't think that's buying you anything.

Reply:
`SwiftPMPackageDescription`?

Reply:
I changed everything to have the suffix `PackageDescription` which seems fine to me now.